src/main/java/dao/PieceDao.java

```diff
@@ -0,0 +1,129 @@
+package dao;
+
+import domain.position.Position;
+import domain.unit.Team;
+import domain.unit.UnitType;
+import entity.Piece;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PieceDao {
+    private static final String SERVER = "localhost:13306"; // MySQL 서버 주소
+    private static final String DATABASE = "janggi"; // MySQL DATABASE 이름
+    private static final String OPTION = "?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC";
+    private static final String USERNAME = "root"; //  MySQL 서버 아이디
+    private static final String PASSWORD = "root"; // MySQL 서버 비밀번호
+
+    public Connection getConnection() {
+        try {
+            return DriverManager.getConnection("jdbc:mysql://" + SERVER + "/" + DATABASE + OPTION, USERNAME, PASSWORD);
+        } catch (final SQLException e) {
+            System.err.println("DB 연결 오류:" + e.getMessage());
+            e.printStackTrace();
+            return null;
```

Comment:
connection을 얻지 못했다면 빠르게 예외를 던져주는게 불필요한 예외가 전파되지 않아서 더 디버깅하기가 수월할 것 같아요. 🤔

(예외 타입은 적절한걸 고민해보시면 좋을 것 같아요. 😃)
```suggestion
            throw RuntimeException("DB 연결 오류:" + e.getMessage());
```